sway/input/seat.c

```diff
@@ -1230,8 +1230,10 @@ void seat_set_focus_surface(struct sway_seat *seat,
 
 	if (surface) {
 		seat_keyboard_notify_enter(seat, surface);
+		sway_input_method_relay_set_focus(&seat->im_relay, surface);
```

Comment:
Nit: for symmetry, this line can go above `seat_tablet_pads_notify_enter`, which also has internal logic for handling the `surface = NULL` case. Then the second diff line below becomes unnecessary.